### DISCUSSION TITLE
Mute/Unmute functions are reverted

### DISCUSSION THREAD

**Discussion by doktoil-makresh** [MonitorControl/MonitorControl#1296] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

My monitor is muted when sound is set to any value but muted
My monitor is not unmuted when sound is muted

### Steps to reproduce

On a Monoprice V3L6W monitor, connected with display port on Macbook Pro 2021 USB-C, install MonitorControl
Then, turn sound volume up -> no sound, while volume icon shows it is not muted
turn sound volume down until muted, or use mute button -> sound is back, while volume icon show it is muted

### Expected behavior

At least, sound is muted when volume is set to 0 (aka muted), and sound is working when volume is set to higher than 0
In the best case scenario, volume control allows to set volume to different value (the reason I've installed this software)

### Anything else?

my display signal goes through a On a Monoprice V3L6W monitor, connected with display port on Macbook Pro 2021 USB-C (more details available here: https://www.amazon.fr/Cable-Matters-DisplayPort-Adaptateur-Thunderbolt/dp/B06ZY1ZJDQ/ref=pd_bxgy_img_sccl_1/262-8434534-7420027?pd_rd_w=kxlHI&content-id=amzn1.sym.3e903e31-26aa-46b7-bc9c-c2062226ae30&pf_rd_p=3e903e31-26aa-46b7-bc9c-c2062226ae30&pf_rd_r=K8660XDV513549FP0JAH&pd_rd_wg=aIhiU&pd_rd_r=0aec36ec-afd9-4ae8-bcbb-2e5d5c7feafe&pd_rd_i=B06ZY1ZJDQ&psc=1)

### Environment Information (please complete the following information)

```markdown
- macOS version: Ventura 13.1
- Mac model:MacBook Pro 16 2021
- MonitorControl version: 4.1.0
- Monitor(s):Monoprice V3L6W
- Apple Silicon/M1 (yes or no): Apple M1 Max
```

**Comment by waydabber** [comment]
Hi there - this is a highly monitor specific problem. Probably the MUTE On/Off values are wired in a reversed fashion in the display firmware.

Do you have the Mute command enabled in advanced settings for the display or is this how it works by default? If the latter, then the best thing you can probably do is not to use DDC volume controls for this display I am afraid or seek an app that does software based volume control (eqMac or some alternative).

**Comment by doktoil-makresh** [comment]
I'm not sure it helps, but everything is working fine on the same monitor using Windows 10/11/Ubuntu 20.04

Regarding DDC, if I disable DDC volume control or DDC mute, then there is no sound whatever I do.
